src/System.Text.Json/tests/Serialization/CacheTests.cs

```diff
@@ -124,28 +124,28 @@ public static void MultipleTypes(ITestClass testObj)
             testObj.Verify();
             string json = JsonSerializer.Serialize(testObj, type);
 
-            void Serialize()
-            {
-                ITestClass localTestObj = (ITestClass)Activator.CreateInstance(type);
-                localTestObj.Initialize();
-                localTestObj.Verify();
-                string json = JsonSerializer.Serialize(localTestObj, type, s_options);
-            };
+            //void Serialize()
```

Comment:
Another TODO? (commented out and modified test) These cache tests are critical

Reply:
SImple test repro. Reverted, and the tests pass.